src/test/kotlin/lotto/generator/MockLottoShop.kt

```diff
@@ -0,0 +1,10 @@
+package lotto.generator
+
+import lotto.domain.LottoTickets
+import lotto.generator.ticket.ManualTicketGenerator
+
+class MockLottoShop(private val manualTicketGenerator: ManualTicketGenerator) : LottoShop {
+    override fun provideAutoTickets(ticketCount: Int, preGeneratedTicketCount: Int): LottoTickets = LottoTickets(listOf())
+    override fun provideManualTickets(manualNumbersList: List<List<Int>>): LottoTickets =
+        manualTicketGenerator.create(manualNumbersList)
```

Comment:
해당 Mock객체는 정상적인 동작을 한다고 보기는 힘들지 않을까요?
fake객체라면 원하는대로 활용할수있는구조로 작성해봐도 좋을거같아요

추상화를 한 이유와 Fake객체를 사용하는 이유가 무엇일지 고민해보면 어떨까요?
테스트가능한 구조를 구민보아요
```suggestion
class MockLottoShop(
     private val autoTickets: LottoTickets,
     private val manualTicket: LottoTickets,
) : LottoShop {
    override fun provideAutoTickets(ticketCount: Int, preGeneratedTicketCount: Int): LottoTickets = autoTickets
    override fun provideManualTickets(manualNumbersList: List<List<Int>>): LottoTickets = manualTicket
```